docs/advanced_topics/customisation/custom_user_models.rst

```diff
@@ -11,7 +11,8 @@ Create a custom user model. In this case we extend the ``AbstractUser`` class an
 two fields. The foreign key references another model (not shown).
 
 .. code-block:: python
-
+  from django.contrib.auth.models import AbstractUser
```

Comment:
Need an extra blank line above this to be valid RestructuredText syntax.

Reply:
Thanks